tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -703,10 +742,6 @@ private MissingStatus recordMissingChangedEtc(String path,
                 problems.add(Choice.missingCoverage);
                 vc.problemCounter.increment(Choice.missingCoverage);
             }
-            if (SubmissionLocales.pathAllowedInLimitedSubmission(path)) {
-                problems.add(Choice.englishChanged);
-                vc.problemCounter.increment(Choice.englishChanged);
-            }
```

Comment:
this block seems to have been left here by mistake; the following block handles englishChanged more carefully

I noticed the bug while working on this ticket -- notice that Choice.englishChanged was added even when choices.contains(Choice.englishChanged) == false, and without checking for all the other necessary conditions

Reply:
I just made a new ticket for this English Changed bug; I think it's actually fixed by this PR: https://unicode-org.atlassian.net/browse/CLDR-15675